JoinColony/colonyCDapp
# fix: motion data, action name, objective desc length

## Description

- Fixed manage colony objective completed action

## Testing

- Create a new objective action in a colony.
- Try to add a description to the objective action.
- You are limited to only 200 characters.
- Complete the action.
- Notice the missing fields in the completed state.

Resolves [https://github.com/JoinColony/colonyCDapp/issues/2242](https://github.com/JoinColony/colonyCDapp/issues/2242)
